pkg/kosmosctl/floater/doctor.go

```diff
@@ -0,0 +1,317 @@
+package floater
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	ctlutil "k8s.io/kubectl/pkg/cmd/util"
+
+	"github.com/kosmos.io/kosmos/pkg/kosmosctl/floater/command"
+	"github.com/kosmos.io/kosmos/pkg/kosmosctl/floater/netmap"
+	"github.com/kosmos.io/kosmos/pkg/kosmosctl/util"
+	"github.com/kosmos.io/kosmos/pkg/version"
+)
+
+type CommandDoctorOptions struct {
+	Namespace          string
+	ImageRepository    string
+	ImageRepositoryDst string
+	Version            string
+
+	Protocol    string
+	PodWaitTime int
+	Port        string
+	HostNetwork bool
+
+	SrcKubeConfig  string
+	DstKubeConfig  string
+	HostKubeConfig string
+
+	SrcFloater *Floater
+	DstFloater *Floater
+}
+
+type Protocol string
+
+const (
+	TCP  Protocol = "tcp"
+	UDP  Protocol = "udp"
+	IPv4 Protocol = "ipv4"
+)
+
+type PrintData struct {
+	command.Result
+	SrcNodeName string
+	DstNodeName string
+	TargetIP    string
+}
+
+func NewCmdDoctor() *cobra.Command {
+	o := &CommandDoctorOptions{
+		Version: version.GetReleaseVersion().PatchRelease(),
+	}
+	cmd := &cobra.Command{
+		Use:                   "dr",
+		Short:                 "Dr.link is an one-shot kubernetes network diagnose tool.",
+		Long:                  "",
+		Example:               "",
+		SilenceUsage:          true,
+		DisableFlagsInUseLine: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctlutil.CheckErr(o.Complete())
+			ctlutil.CheckErr(o.Validate())
+			ctlutil.CheckErr(o.Run())
+			return nil
+		},
+		Args: func(cmd *cobra.Command, args []string) error {
+			for _, arg := range args {
+				if len(arg) > 0 {
+					return fmt.Errorf("%q does not take any arguments, got %q", cmd.CommandPath(), args)
+				}
+			}
+			return nil
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVarP(&o.Namespace, "namespace", "n", util.DefaultNamespace, "Kubernetes namespace.")
+	flags.StringVarP(&o.ImageRepository, "image-repository", "r", "ghcr.io/kosmos-io/clusterlink", "Image repository.")
```

Comment:
modify default repo to ghcr.io/kosmos-io

Reply:
ok~I'm done